app/lib/parachain_manager.py

```diff
@@ -137,7 +139,7 @@ def get_permanent_slot_lease_period_length(substrate_client):
     return substrate_client.get_constant("AssignedSlots", "PermanentSlotLeasePeriodLength").value
 
 
-def initialize_parachain(substrate_client, sudo_seed, para_id, state, wasm, lease_period_count=0):
+def initialize_parachain(substrate_client, sudo_seed, para_id, state, wasm, lease_period_count=0, force_queue_action=True):
```

Comment:
` force_queue_action` already defaults to True in the router. It might be better not to set a default value.

Reply:
I removed default form `onboard_parachain_by_id` function